src/status_im/contexts/chat/events.cljs

```diff
@@ -156,8 +156,10 @@
                  :effects.async-storage/set {:chat-id nil
                                              :key-uid nil}}
                 (link-preview/reset-all)
-                (delete-for-me/sync-all)
-                (delete-message/send-all)
+                (fn [cofx]
+                  (delete-for-me/sync-all cofx))
+                (fn [cofx]
+                  (delete-message/send-all cofx))
```

Comment:
* Here we're wrapping these functions because `delete-for-me/sync-all` and `delete-message/send-all` are no longer defined with `rf/defn`. 
  * It seems by default that functions defined with `rf/defn` are composable with `rf/merge` (when we don't pass all the arguments I think), and `rf/merge` likes to compose these functions by passing a `cofx` object. 
  * So instead we just manually wrap the function, which seems to have the same behaviour as before.